gadgets/trace_dns/program.bpf.c

```diff
@@ -11,6 +11,7 @@
 #include <bpf/bpf_helpers.h>
 #include <bpf/bpf_endian.h>
 
+#include <gadget/buffer.h>
```

Comment:
Should we define `MAX_EVENT_SIZE` in all these gadgets to avoid overallocating memory?

Reply:
The heap map uses `MAX_EVENT_SIZE` and it's defined to be 10240 bytes by default. 

https://github.com/inspektor-gadget/inspektor-gadget/blob/595261b6daa06e1ecb737d44ae09812e0e595c9e/include/gadget/buffer.h#L22-L27

I think 10240 is too much and waste space for most of the cases, so I'd suggest to define it on each gadget to be equal to the size of the event.